src/flow/kishimoto.jl

```diff
@@ -0,0 +1,75 @@
+# Method when using Boykov-Kolmogorov as a subroutine
+# Kishimoto algorithm
+function kishimoto{T<:AbstractFloat}(
+  flow_graph::DiGraph,                       # the input graph
+  source::Int,                               # the source vertex
+  target::Int,                               # the target vertex
+  capacity_matrix::AbstractArray{T,2},       # edge flow capacities
+  flow_algorithm::BoykovKolmogorovAlgorithm, # keyword argument for algorithm
+  routes::Int                                # keyword argument for routes
+  )
+  # Initialisation
+  flow, F, labels = maximum_flow(flow_graph, source, target,
+         capacity_matrix, algorithm = flow_algorithm)
+  restriction = flow / routes
+  flow, F, labels = maximum_flow(flow_graph, source, target, capacity_matrix,
+         algorithm = flow_algorithm, restriction = restriction)
+
+  # Loop
+  i = 1
+  while flow ≉ routes * restriction && flow < routes * restriction
```

Comment:
Ok, the good point is that I found (and corrected) a small bug when testing for the precision.

The bad news is that the compute of a 3-route flow on the first graph (8 vertexes) in `test/flow/multiroute_flow.jl` is enough to bring up the problem of the float precision.
It is not totally surprising since any of the algorithm can call a maximum_flow function with a restriction of, for instance, 1/3.

I don't see a good way out of it, but I am ready to chase after any good idea :)

So, I will update the PR for now (coverage for multiroute flow should be complete).

Thank you again for all the comments and advices